lib/commands/implementations/breaking-news.js

```diff
@@ -0,0 +1,59 @@
+const moment = require('moment');
+const _ = require('lodash');
+
+const Command = require('../command-interface');
+const CommandOutput = require('../command-output');
+const { makeMute } = require('../../chat-utils/punishment-helpers');
+const parseDurationToSeconds = require('../../chat-utils/duration-parser');
+const formatDuration = require('../../chat-utils/format-duration');
+
+function breakingNews(defaultMaxAge) {
+  let enabled = false;
+
+  return (input, services, rawMessage) => {
+    const matched = /(\d+[HMDShmds])?/.exec(input);
+    const maxAgeStr = _.get(matched, 1, defaultMaxAge).toLowerCase();
+
+    enabled = !enabled;
```

Comment:
Consider accepting on/off as parameter to the command so it matches behavior of !mutelinks.